### ISSUE TITLE
Minio endpoint check is failing on Python 3.7.6

### ISSUE THREAD

**Issue by kevinappy** [minio/minio-py#837]
It seems that `urllib.parse.urlsplit` was changed in the Python 3.7.6 and this breaks the behavior of the latest release versions of Minio.

```
Python 3.7.6 (default, Dec 28 2019, 05:50:43) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from urllib.parse import urlsplit
>>> urlsplit("localhost:9000")
SplitResult(scheme='localhost', netloc='', path='9000', query='', fragment='')
>>> urlsplit("localhost")
SplitResult(scheme='', netloc='', path='localhost', query='', fragment='')
>>> urlsplit("//localhost:9000")
SplitResult(scheme='', netloc='localhost:9000', path='', query='', fragment='')
>>> from minio.helpers import is_valid_endpoint
>>> is_valid_endpoint("localhost:9000")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/minio/helpers.py", line 303, in is_valid_endpoint
    raise InvalidEndpointError('Hostname cannot have a scheme.')
minio.error.InvalidEndpointError: InvalidEndpointError: message: Hostname cannot have a scheme.
>>> is_valid_endpoint("//localhost:9000")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/minio/helpers.py", line 316, in is_valid_endpoint
    raise InvalidEndpointError('Hostname does not meet URL standards.')
minio.error.InvalidEndpointError: InvalidEndpointError: message: Hostname does not meet URL standards.
>>> is_valid_endpoint("localhost")
True
```

The behavior before 3.7.6 was that `localhost:9000` was a valid endpoint.

**Comment by harshavardhana** [comment]
Fixed in #836

**Comment by harshavardhana** [comment]
and released 5.0.6
